src/main/kotlin/no/nav/klage/oppgave/clients/azure/DefaultAzureGateway.kt

```diff
@@ -100,7 +100,7 @@ class DefaultAzureGateway(private val microsoftGraphClient: MicrosoftGraphClient
         }
 
     private fun mapToEnhet(enhetNr: String): Enhet =
-        KodeverkEnhet.values().find { it.navn == enhetNr }
+        no.nav.klage.kodeverk.Enhet.entries.find { it.navn == enhetNr }
```

Comment:
Her er det laget et alias for å "slippe" å fullkvalifisere, men greit å bytte ut til dette hvis det ikke leser like bra. Men da bør også importen fikses på.

Reply:
Stemmer, skjedde et par ting her. Jeg bruker aliaset heller.